source/tests/yats-test/util_test.cpp

```diff
@@ -5,46 +5,29 @@
 #include <yats/Util.h>
 
 
-TEST(is_unique_ptr_test, util_test)
+TEST(util_test, is_unique_ptr_test)
 {
 	ASSERT_TRUE(is_unique_ptr_v<std::unique_ptr<int>>);
 	ASSERT_FALSE(is_unique_ptr_v<std::shared_ptr<int>>);
 	ASSERT_FALSE(is_unique_ptr_v<int>);
+	ASSERT_FALSE(is_unique_ptr_v<int*>);
+	ASSERT_FALSE(is_unique_ptr_v<int&>);
+	ASSERT_FALSE(is_unique_ptr_v<const int&>);
+	ASSERT_FALSE(is_unique_ptr_v<void>);
```

Comment:
Could you add one test case that is not a literal. So something like `is_unique_ptr_v<MyClass*>`